06.bowling_object/game.rb

```diff
@@ -0,0 +1,43 @@
+require './frame'
```

Comment:
```suggestion
require './frame'

```

requireの下は改行することが多いかも